Modules/Devices/src/Devices.Infrastructure/PushNotifications/DirectPush/DirectPnsCommunicationOptions.cs

```diff
@@ -37,7 +37,7 @@ public bool HasConfigForAppId(string appId)
             return serviceAccountForServiceAccountNameExists;
         }
 
-        public string GetServiceAccountForAppId(string appId)
+        public string GetServiceAccountForAppId(string? appId)
```

Comment:
Why did you make the appId nullable? As far as I can see from the usage it is never null.